spec/models/user_spec.rb

```diff
@@ -46,6 +46,42 @@
     end
   end
 
+  context 'Creating user with user name test_student' do
```

Comment:
you can replace most of this with calls to:

```ruby
it { is_expected.to validate_exclusion_of(:user_name).in_array([User::TEST_STUDENT_USER_NAME]) }
```

and just change the `User.type` of the [`subject`](https://github.com/thoughtbot/shoulda-matchers#on-the-subject-of-subject)

Reply:
If you do this, also move it to the top of the file with the rest of the validation tests.